src/status_im/contexts/profile/edit/modal/style.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.contexts.profile.edit.modal.style
+  (:require
+    [quo.foundations.colors :as colors]
+    [react-native.platform :as platform]))
+
+(def continue-button
+  {:width        "100%"
+   :margin-left  :auto
+   :margin-top   (if platform/android? :auto 0)
+   :margin-right :auto})
+
+(def button-container
+  {:width         "100%"
+   :padding-left  20
+   :padding-right 20
+   :padding-top   12
+   :align-self    :flex-end
+   :height        64})
```

Comment:
I'm not certain but I think we tend to avoid using "100%" (or percentages in general) for things like width or height. Is it possible we can achieve something similar by using flexbox via `flex-direction: row` or similar?

Reply:
I agree, I don't exactly know why this has been written like this, but I think we may revisit this implementation, as said in the description I cleaned a bit the namespace, extended the implementation but didn't try to rework it.

BTW, @ilmotta if this floating button were used in other places, IMO it's worth rewriting it in reanimated, to make it look more soft and responsive. We could address it along with the switch from `ratoms` -> `use-state`. I believe it's a  low prio.